sourceme.sh

```diff
@@ -0,0 +1 @@
+export $(grep -v '^#' .env | xargs -0)
```

Comment:
Put this file in your git ignore so that it doesn't get pushed to the repository.